recipes/aaf/all/conanfile.py

```diff
@@ -84,9 +85,12 @@ def package(self):
     def package_info(self):
         if self.settings.os == "Windows":
             suffix = "D" if self.settings.build_type == "Debug" else ""
-            self.cpp_info.libs = [f"AAF{suffix}", f"AAFIID{suffix}", "AAFCOAPI"]
+            self.cpp_info.libs = [f"AAF{suffix}", f"AAFIID{suffix}"]
+            # The static library loads a DLL at runtime, on Windows it needs to be able
+            # to find it in PATH, see https://aaf.sourceforge.net/AAFProjectFAQ.html
+            self.runenv_info.prepend_path("PATH", os.path.join(self.package_folder, "bin"))
```

Comment:
is it not supposed to be already defined by conan since `cpp_info.bindirs = ["bin"]`?